content/blog/eng/operators-overview.md

```diff
@@ -0,0 +1,140 @@
+
+
+# Precedence
+The Operator precedence (aka operator binding) is a collection of rules that reflect conventions about which procedures perform first in order to evaluate a given expression. This conventions are very useful to eliminate ambiguity while we perform an expression with a group of operators.
+
+# Operand coercion
+Some languages also allow to the operands of an operator to be implicitly converted, or coerced, to suitable data types for the operation to occur. 
+For example, in Perl coercion rules lead into 12 + "3.14" producing the result of 15.14. The text "3.14" is converted to the number 3.14 before addition can takes place. Further, 12 is an integer and 3.14 is either a floating or fixed-point number (a number that has a decimal place in it) so the integer is then converted to a floating point or fixed-point number respectively.
+JavaScript follows opposite rules—finding the same expression above, it will convert the integer 12 into a string "12", then concatenate the two operands to form "123.14".
+In the presence of coercions in a language, the programmer must be aware of the specific rules regarding operand types and the operation result type to avoid subtle programming mistakes.
+
+
+# Common operators
+
+# Access operator
+Access operator . is used for to access a member of a namespace or a type (a.b). For example we can use access operator to access a nested namespace  String.Indices or to access type members static and non-static.
+
+# Scope resolution operator
+The scope resolution operator ::, helps to identify and specify the context which an identifier refers.
+
+# Indexer operator
+The indexer operator [ ] is typically used to access a variable in a sequence or in a pointer element access. 
+
+# Null-conditional operators
+Null-conditional operators ?. or ?[ ]  allow access to a member only if that operand evaluates to non-null; otherwise, it returns null.
+
+# Compound Assignment Operators
+Compound Assignment operators provide a shorter syntax for assigning the result for an arithmetic and bitwise operator. They perform the operation on the two operands before assigning the result to the first operand.
+For example  x += 2  is a shortcut for  x = x + 2.
+
+# Invocation operator
+The invocation operator () call a method or invoke a delegate.
+
+# Relational operator
+The relational operator are constructs that define a type of relationship between two entities, this definition includes both equality and inequality.
+An expression built with relational operators is called conditional expression and expressions of this type are defined predicates. If the language support the Boolean type (es. Java, PHP) the relational operator returns a true/false value, instead in other languages like C the relational operator returns 0 or 1, where the 0 correspond to false value and 1 to the true value.
```

Comment:
Is not clear the first phrase.

Reply:
Can you explain to me better what is not clear?

Reply:
What I understand is:  
conditional expression = an expression built with relational operators  
predicates = a set of conditional expressions

Is it right?

Reply:
Yes, that's exactly what I meant.